GiniVisionExample/GiniVisionExampleTests/GiniVisionExampleTests-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```

Comment:
Why did you add a Bridging header to the project? We don't use Objective-C.

Reply:
I thought the same, but when I created the test for the first time Xcode asked to have one but we can remove it yep